wpilib/controller/__init__.py

```diff
@@ -3,22 +3,22 @@
 # autogenerated by 'robotpy-build create-imports wpilib.controller wpilib.controller._controller'
 from ._controller import (
     ArmFeedforward,
+    ElevatorFeedforward,
     ElevatorFeedforwardFeet,
     ElevatorFeedforwardMeters,
     PIDController,
     ProfiledPIDController,
     RamseteController,
+    SimpleMotorFeedforward,
     SimpleMotorFeedforwardFeet,
     SimpleMotorFeedforwardMeters,
 )
 
 __all__ = [
     "ArmFeedforward",
-    "ElevatorFeedforwardFeet",
```

Comment:
Should still be in all.

Reply:
Do we really care about documenting these? I don't think we want to actively encourage users to use them if we're going to remove them next season.